src/modules/drawing/components/DrawingStylePopup.vue

```diff
@@ -13,7 +13,7 @@
                     <label for="text">{{ $t('draw_popup_title_annotation') }}:</label>
                     <textarea id="text" v-model="text" class="form-control" rows="1"></textarea>
                 </div>
-                <div class="form-group">
+                <div v-if="feature.get('type') !== 'TEXT'" class="form-group">
```

Comment:
as `feature.get('type')` is used more than once in the template, it would be more efficient to have a `computed` with it, so that it is calculated once (each time feature changes) and not 4 or 5 times in the template.

That's usually a good rule of thumb, do not place too much data access logic in the template, but prefer a `computed` properties for that.